general/lecture1/tasks/_7/Task7.java

```diff
@@ -0,0 +1,45 @@
+package general.lecture1.tasks._7;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Write a program that prints all odd numbers from N to 1.
+ * N should be read from the input.
+ * Every number should be printed on the new line.
+ * Create two versions of the program: one should use the for loop,
+ * second - the while loop.
+ **/
+public class Task7 {
+    Scanner sc = new Scanner(System.in);
+    int number = sc.nextInt();
```

Comment:
the number could be passed as a method argument to both versions, you don't need to define it as instance variables

Reply:
I did it for the better code readability

Reply:
ok, in such case it would be better to pass it in the constructor - reading data from standard input in such way can cause unpredictable results

Reply:
Okay, clear. Thank you for the comments, will work on it.